apps/web/package.json

```diff
@@ -40,43 +46,28 @@
     ]
   },
   "dependencies": {
-    "@babel/cli": "7.25.7",
-    "@babel/core": "7.25.8",
-    "@babel/eslint-parser": "^7.25.8",
-    "@babel/node": "7.25.7",
-    "@babel/plugin-proposal-class-properties": "^7.18.6",
-    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
-    "@babel/plugin-transform-runtime": "^7.25.7",
-    "@babel/preset-env": "7.25.8",
-    "@babel/preset-react": "^7.25.7",
     "@deck.gl-community/editable-layers": "^9.0.3",
     "@deck.gl-community/layers": "^9.0.3",
-    "@deck.gl/core": "9.0.33",
-    "@deck.gl/extensions": "^9.0.33",
-    "@deck.gl/geo-layers": "9.0.33",
-    "@deck.gl/layers": "9.0.33",
-    "@deck.gl/mapbox": "^9.0.33",
-    "@deck.gl/mesh-layers": "9.0.33",
-    "@deck.gl/react": "9.0.33",
+    "@deck.gl/core": "9.0.35",
+    "@deck.gl/extensions": "^9.0.35",
```

Comment:
did you want to remove the ^ here, just to make sure the deckgl versions issue doesn't return?

Reply:
sure